install_deps.sh

```diff
@@ -100,13 +104,13 @@ echo 'Freetype installation is finished.'
 
 echo 'Installing ICU.'
 
-rm -rf icu4c-62_1-src
+rm -rf "${ICU_DIR_NAME}"
```

Comment:
I think this was actually meant to delete the `icu` directory (the one after extracting the archive).